draft-ietf-quic-qpack.md

```diff
@@ -192,7 +192,7 @@ while the decoder is relatively simple.
 
 An encoder MUST ensure that a header block which references a dynamic table
 entry is not processed by the decoder after the referenced entry has been
-evicted.  Hence the encoder needs to track information about each compressed
```

Comment:
Once a header block is sent, this collection of information does not change.  This makes me believe "retain" is a more fitting word than "track".